src/CMakeLists.txt

```diff
@@ -2,6 +2,13 @@ if(HAVE_BFD_DISASM)
   set(BFD_DISASM_SRC bfd-disasm.cpp)
 endif()
 
+if (HARDENED_STDLIB)
+  add_compile_definitions("-D_LIBCPP_HARDENING_MODE=_LIBCPP_HARDENING_MODE_DEBUG")
+  add_compile_definitions("-D_GLIBCXX_ASSERTIONS")
+  add_compile_definitions("-D_FORTIFY_SOURCE=3")
+  add_compile_options("-O1")
+endif()
```

Comment:
seems like this is better placed over here with other compile options: https://github.com/bpftrace/bpftrace/blob/a30d61b56368ef33092fa261c168424168b1bcd3/CMakeLists.txt#L43

Reply:
The `-O1` option was breaking the tests. I'll remove it as you suggested below and move the options to the top level!